cmd/osm-controller/osm-controller.go

```diff
@@ -239,6 +242,19 @@ func main() {
 		"/health/ready": health.ReadinessHandler(funcProbes, getHTTPHealthProbes()),
 		"/health/alive": health.LivenessHandler(funcProbes, getHTTPHealthProbes()),
 	})
+
+	webhookHandlerCert, err := certManager.IssueCertificate(
+		certificate.CommonName(fmt.Sprintf("%s.%s.svc", validatingWebhookServiceCN, osmNamespace)),
+		constants.XDSCertificateValidityPeriod)
+	if err != nil {
+		log.Fatal().Err(err).Msgf("Error issuing certificate for the validating webhook.")
```

Comment:
Please use `events.GenericEventRecorder().FatalEvent` for fatal errors in `main()`.